### PR TITLE
 CI: use default expiration time everywhere

### PR DESCRIPTION
**Pull request by SkySkimmer**
Was: `CI: put expire_in 1 month on ci-template`

### CODE REVIEW

**Comment by SkySkimmer** [comment]
Actually this is a noop https://docs.gitlab.com/ee/user/gitlab_com/index.html#gitlab-cicd
>Artifacts expiry time 
>From June 22, 2020, deleted after 30 days unless otherwise specified (artifacts created before that date have no expiry).
and all the 
~~~
    # Gitlab doesn't support yet "expire_in: never" so we use the instance default
    # expire_in: never
~~~
are meaningless now

**Comment by SkySkimmer** [comment]
changed

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -564,7 +553,6 @@ test-suite:edge:dune:dev:
     paths:
       - _build/log
       - _build/default/test-suite/logs
-    expire_in: 2 week
```
Comment (SkySkimmer) [inline .gitlab-ci.yml:567]: This one was shorter but I don't think 2 more weeks of logs will be what overflows storage

**Comment by JasonGross** [comment]
> Actually this is a noop [docs.gitlab.com/ee/user/gitlab_com/index.html#gitlab-cicd](https://docs.gitlab.com/ee/user/gitlab_com/index.html#gitlab-cicd)
> 
> > Artifacts expiry time
> > From June 22, 2020, deleted after 30 days unless otherwise specified (artifacts created before that date have no expiry).

Maybe we still want to be explicit, though?  Note that this controls how frequently PRs need to be rebased / merges to master need to happen in order to use the bug minimizer.

**Comment by ejgallego** [comment]
@SkySkimmer is it OK for you if we revisit this after #15560 ?

**Comment by ejgallego** [comment]
Not sure what the closure means. I'm OK with either:
- merging this and rebasing the other PR on top
- or merging the other PR and rebasing this on top (which could be possibly empty)

But in any case I think we should keep this open unless you have already determined that you wanna do 2 and the result is an empty commit.

**Pull request closed without merging**